backend/lib/edgehog/containers/deployment.ex

```diff
@@ -0,0 +1,51 @@
+#
+# This file is part of Edgehog.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defmodule Edgehog.Containers.Deployment do
+  @moduledoc false
+  use Edgehog.MultitenantResource,
+    domain: Edgehog.Containers
+
+  actions do
+    defaults [:read, :destroy, create: [:device_id, :release_id]]
+  end
+
+  attributes do
+    uuid_primary_key :id
+
+    timestamps()
+  end
+
+  relationships do
+    belongs_to :device, Edgehog.Devices.Device
+
+    belongs_to :release, Edgehog.Containers.Release do
+      attribute_type :uuid
+    end
```

Comment:
Shouldn't this also include a reference to the containers?
```suggestion
      has_many :containers, Edgehog.Containers.Container
    end
```

Reply:
General note: it's not mandatory to have _all_ `has_many` relationships everywhere, they can be added if/when they're actually needed/used